charts/incubator/kubernetes-reflector/values.yaml

```diff
@@ -0,0 +1,80 @@
+image:
+  # TODO: update image to use TC repo
+  repository: emberstack/kubernetes-reflector
+  pullPolicy: IfNotPresent
+  tag: 7.1.210
+
+env:
+  ES_Serilog__MinimumLevel__Default: Information
+  ES_Reflector__Watcher__Timeout: ""
+  ES_Reflector__Kubernetes__SkipTlsVerify: "false"
```

Comment:
env is no longer on the top level, but under the specific container that belong.
eg 
```yaml
        main:
          env:
            ES_...:
```